packages/cli/src/cmds/preview/depsChecker/funcToolChecker.ts

```diff
@@ -149,6 +166,10 @@ export class FuncToolChecker implements IDepsChecker {
     return path.join(os.homedir(), `.${ConfigFolderName}`, "bin", "func");
   }
 
+  private static getSentinelPath(): string {
+    return path.join(os.homedir(), `.${ConfigFolderName}`, "sentinel");
```

Comment:
The name `sentinel` is too general and it is put directly in the `.fx` folder. Better change to `func-sentinel` or `env-checker-func-sentinel` to avoid confusion or conflicting with future files.

Reply:
Good idea, added